WordPress/Classes/ViewRelated/Blaze Campaigns/BlazeCampaignTableViewCell.swift

```diff
@@ -0,0 +1,196 @@
+
+    private lazy var featuredImageView: CachedAnimatedImageView = {
+        let imageView = CachedAnimatedImageView()
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+
+        NSLayoutConstraint.activate([
+            imageView.widthAnchor.constraint(equalToConstant: Metrics.featuredImageSize),
+            imageView.heightAnchor.constraint(equalTo: imageView.widthAnchor)
+        ])
+
+        imageView.clipsToBounds = true
+        imageView.layer.cornerRadius = Metrics.featuredImageCornerRadius
+        return imageView
+    }()
+
+    // MARK: - Properties
+
+    private lazy var imageLoader: ImageLoader = {
+        return ImageLoader(imageView: featuredImageView, gifStrategy: .mediumGIFs)
+    }()
+
+    // MARK: - Initializers
+
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        commonInit()
+    }
+
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+        commonInit()
+    }
+
+    // MARK: - Public
+
+    func configure(with viewModel: BlazeCampaignViewModel, blog: Blog) {
+        statusView.configure(with: viewModel.status)
+
+        titleLabel.text = viewModel.title
+
+        imageLoader.prepareForReuse()
+        featuredImageView.isHidden = viewModel.imageURL == nil
+        if let imageURL = viewModel.imageURL {
+            let host = MediaHost(with: blog, failure: { error in
+                WordPressAppDelegate.crashLogging?.logError(error)
+            })
+
+            let preferredSize = CGSize(width: featuredImageView.frame.width, height: featuredImageView.frame.height)
+            imageLoader.loadImage(with: imageURL, from: host, preferredSize: preferredSize)
+        }
+
+        configureStatsViews(with: viewModel)
+        configureVoiceOver(with: viewModel)
+    }
+
+    // MARK: - Helpers
+
+    private func commonInit() {
+        setupViews()
+        applyStyle()
+    }
+
+    private func applyStyle() {
+        backgroundColor = .DS.Background.primary
+    }
+
+    private func setupViews() {
+        contentView.addSubview(mainStackView)
+        contentView.pinSubviewToAllEdges(mainStackView)
+    }
+
+    private func configureStatsViews(with viewModel: BlazeCampaignViewModel) {
+
+        var subviews: [UIView] = []
+
+        if viewModel.isShowingStats {
+            let impressionsView = BlazeCampaignSingleStatView(title: Strings.impressions)
+            impressionsView.countString = "\(viewModel.impressions)"
+
+            let clicksView = BlazeCampaignSingleStatView(title: Strings.clicks)
+            clicksView.countString = "\(viewModel.clicks)"
+
+            subviews += [impressionsView, clicksView]
+        }
+
+        let budgetView = BlazeCampaignSingleStatView(title: Strings.budget)
+        budgetView.countString = "\(viewModel.budget)"
+        subviews += [budgetView, UIView()]
+
+        statsStackView.addArrangedSubviews(subviews)
```

Comment:
I think it may be missing the logic that handles cell reuse.

Reply:
Fixed! [eaa3395](https://github.com/wordpress-mobile/WordPress-iOS/pull/20886/commits/eaa339516bd092f90db16175e6054f91ceef1589)